lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+const DEFAULT_MANIFEST_PREFS = {
+  "social.manifest.twitter-com": {
+    "author": "Twitter, Inc.",
+    "description": "Use your Firefox Twitter Share to quickly author tweets about the pages you discover across the web.  We shorten URLs for quick sharing using the t.co link shortener.",
+    "homepageURL": "https://twitter.com",
+    "icon32ULR": "images/twitter/icon32.png",
```

Comment:
heh.  fixed on the activations site (not yet updated).  If it's easier, you can grab the base64 from here:
https://github.com/mixedpuppy/socialapi-directory/blob/production/en-US/twitter.html